xenon/cogs/templates.py

```diff
@@ -40,7 +40,7 @@ def __init__(self, bot):
     @cmd.cooldown(1, 30, cmd.BucketType.user)
     async def create(self, ctx, backup_id=None, name=None, *, description=None):
         """
-        Turn a private backup into a PUBLIC template.
+        Turn a private backup into a **PUBLIC** template.
```

Comment:
Making it even HUGER than it already is :100:
What about also disabling caps lock here? Like `**public**`

Reply:
It's that huge for a reason. Many people don't get that templates are public and try to use it to load backups from different accounts.